src/main/java/com/jwplayer/jwplatform/resource/Custom_Params.java

```diff
@@ -0,0 +1,5 @@
+package com.jwplayer.jwplatform.resource;
+
+public class Custom_Params {
```

Comment:
I don't see these resource classes being used.

Reply:
So, the idea is that from the api call response, we can either just return a JSONObject or actually map it to the resource object so it's more user friendly. So it might look like response = Unirest.post(requestUrl).asObject(ObjectClass); instead of response = Unirest.post(requestUrl).asJson();. Is that a good idea?

Reply:
I'm yet to add more to this class. But didn't want to incase we don't want to do it like this.

Reply:
What did you have in mind for the class design for these resources?

Reply:
It would follow the developer reference where the object and its fields are specified. Does that makes sense?